### ISSUE TITLE
about result

### ISSUE THREAD

**Issue by liujiang137** [LW-Ricarido/Emotion-Recognize#1]
I want to know if your accuracy is so high because you have modified the code, I can't reproduce your results. My accuracy can't be improved by 38%.

**Comment by liujiang137** [comment]
There is a problem with this code of 'DLP_LOSS' ，The value of the loss function cannot be lowered。But when I change the loss function to a crossentropy loss function, the accuracy can reach 72.458.

**Comment by YingjianLi** [comment]
I have the same problem. @liujiang137 , the acc stays at 38.625%, and doesn't change anymore. did you fix this?

**Comment by liujiang137** [comment]
> I have the same problem. @liujiang137 , the acc stays at 38.625%, and doesn't change anymore. did you fix this?

Now I can only judge that there is a problem with the code of the DLP_Loss. What do you think will be the problem?

**Comment by LW-Ricarido** [comment]
> There is a problem with this code of 'DLP_LOSS' ，The value of the loss function cannot be lowered。But when I change the loss function to a crossentropy loss function, the accuracy can reach 72.458.

I do not think there is a problem with my code of 'DLP_LOSS'. The true problem is that I didn't update my training script. Sorry for that. 
I had faced the same problem at first that loss cannot be lowered. **You need to set param lam small enough.**
There is my trainning script. You can try with this. If this script does not work, feel free to contact with me.

python3 -u main.py \    
    -shuffle \ 
    -train_record \
    -model DLP_CNN \
    -data_dir ../DataSet/EmotioNet \
    -train_list ../DataSet/EmotioNet/train_set \
    -test_list ../DataSet/EmotioNet/val_set \
    -save_path checkpoints/EmotioNet \
    -output_classes 16 \
    -pretrain checkpoints/EmotioNet/DLP_CNN_Baseline/model_23.pth\
    -n_epochs 40 \
    -learn_rate 0.001 \
    -batch_size 64 \
    -k 20 \
    -lam 0.0005 \
    -criterion DLP_LOSS \
    -workers 2 \
    -nGPU 1 \
    -decay 30 \
    -size 100 \
    -ckpt 23 \
    -resume \
2>&1 | tee train_DLP_CNN_EmotioNet.log

**Comment by Jiangjiang2018** [comment]
@LW-Ricarido I try with the small lam as your training script provides,but it still dose not work and the acc stays at 38%-39% ,so what is the problem?

**Comment by LW-Ricarido** [comment]
> @LW-Ricarido I try with the small lam as your training script provides,but it still dose not work and the acc stays at 38%-39% ,so what is the problem?

Could you provide some details of your training script? And what dataset did you use?
And I will run my own training script today to double check.

**Comment by Jiangjiang2018** [comment]
> @LW-Ricarido I try with the small lam as your training script provides,but it still dose not work and the acc stays at 38%-39% ,so what is the problem?
> 
> Could you provide some details of your training script? And what dataset did you use?
> And I will run my own training script today to double check.


I changed the lr to 0.01 and it can work  now , this is my  trainning script.

python3 -u main.py \
    -shuffle \
    -train_record \
    -model DLP_CNN \
    -data_dir DataSet/RAF/basic/Image/aligned/ \
    -train_list DataSet/RAF/basic/train_set \
    -test_list DataSet/RAF/basic/validation_set \
    -label_list DataSet/RAF/basic/ \
    -save_path checkpoints \
    -output_classes 7 \
    -n_epochs 40 \
    -learn_rate 0.01 \
    -batch_size 64 \
    -k 20 \
    -lam 0.0005 \
    -criterion DLP_LOSS \
    -workers 0 \
    -nGPU 2 \
    -decay 30 \
    -size 100 \
2>&1 | tee train_DLP_CNN.log

**Comment by LW-Ricarido** [comment]
> > @LW-Ricarido I try with the small lam as your training script provides,but it still dose not work and the acc stays at 38%-39% ,so what is the problem?
> > Could you provide some details of your training script? And what dataset did you use?
> > And I will run my own training script today to double check.
> 
> I changed the lr to 0.01 and it can work now , this is my trainning script.
> 
> python3 -u main.py 
> -shuffle 
> -train_record 
> -model DLP_CNN 
> -data_dir DataSet/RAF/basic/Image/aligned/ 
> -train_list DataSet/RAF/basic/train_set 
> -test_list DataSet/RAF/basic/validation_set 
> -label_list DataSet/RAF/basic/ 
> -save_path checkpoints 
> -output_classes 7 
> -n_epochs 40 
> -learn_rate 0.01 
> -batch_size 64 
> -k 20 
> -lam 0.0005 
> -criterion DLP_LOSS 
> -workers 0 
> -nGPU 2 
> -decay 30 
> -size 100 
> 2>&1 | tee train_DLP_CNN.log

Since my code can work with this script, I will close this issue.
